hexa/static/js/hexa.js

```diff
@@ -423,3 +423,16 @@ function AdvancedSearch(initialQuery) {
         }
     }
 }
+
+/**
+ * 
+ * Display text property secret
+ */
+function SecretSection() {
+    return {
+        secret: true,
+        showSecret($event) {
```

Comment:
Based on the implementation, it's more a `toggleSecret` than a `showSecret` since it handles the two use cases.

Reply:
Yes, will change it.